index.html

```diff
@@ -14,14 +14,92 @@
   <![endif]-->
 </head>
 
-<body>
+<body { 
+  background-color: rgba(0,0,0,1.00);
```

Comment:
Not exactly sure what you're trying to do here? Maybe DM me about it on Slack

Reply:
I forgot to delete that. I was trying to change the background of the page to black, but realized I needed to do it in style sheets. I was just tunnel-visioned on HTML cause of bootstrap.